environment.yml

```diff
@@ -15,36 +11,20 @@ dependencies:
     - joblib==0.10.3
     - mako
     - ipywidgets
-    - numba
     - flask
```

Comment:
Where do we need flask?

Reply:
It's used in file rllab/viskit/frontend.py. That scripts runs the following website
![image](https://user-images.githubusercontent.com/35857569/40073609-254a1bd4-582c-11e8-9072-7d6f0e54f604.png)
It seems you can plot graphs from EPS files from there.